src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -122,14 +119,11 @@ const BioAndDMSetting: React.FC<BioAndDMSettingProps> = ({
           onPress={toggleSwitchAnonAllowFollowing}
           style={styles.toggleSwitchAnonFollowing}>
           <ToggleSwitch
-            isOn={isAllowFollowingSendDM}
-            onToggle={toggleSwitchAnonAllowFollowing}
-            onColor={'#9DEDF1'}
-            label={'Only allow anon DMs from users you follow?'}
-            offColor="#F5F5F5"
-            circleColor={isAllowFollowingSendDM ? '#00ADB5' : colors.white}
-            size="small"
-            labelStyle={styles.toggleLabelFollowingDM}
+            value={isAllowFollowingSendDM}
+            onValueChange={toggleSwitchAnonAllowFollowing}
+            labelLeft="Only allow anon DMs from users you follow?"
+            circleActiveColor={colors.blue1}
+            activeTextColor={colors.blue1}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the properties passed to the `ToggleSwitch` component have been updated. Verify that these changes are correct and do not introduce any issues.
```diff
-            isOn={isAllowFollowingSendDM}
-            onToggle={toggleSwitchAnonAllowFollowing}
-            onColor={'#9DEDF1'}
-            label={'Only allow anon DMs from users you follow?'}
-            offColor="#F5F5F5"
-            circleColor={isAllowFollowingSendDM ? '#00ADB5' : colors.white}
-            size="small"
-            labelStyle={styles.toggleLabelFollowingDM}
+            value={isAllowFollowingSendDM}
+            onValueChange={toggleSwitchAnonAllowFollowing}
+            labelLeft="Only allow anon DMs from users you follow?"
+            circleActiveColor={colors.blue1}
+            activeTextColor={colors.blue1}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->